### PR TITLE
Feature/email setup

### PR DESCRIPTION
**Pull request by basantrijal**
email setup with smtp configs in development.rb file and sidekiq.

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class UserMailer < ApplicationMailer
+
+  def registration_success(user)
+    @user = user
+    mail(to: @user.email, subject: 'Welcome to MemryBook')
```
Comment (allanxavierm) [inline app/mailers/user_mailer.rb:5]: No need to assign the parameter to a instance variable, you can straightaway use the parameter in the method. Please define message in en.yml

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  end
+
+  def forgot_password(user)
+    @user = user
```
Comment (allanxavierm) [inline app/mailers/user_mailer.rb:9]: Again here no need of assigning it to a instance variable.

File: config/environments/development.rb [hunk 3]
```diff
@@ -40,4 +40,20 @@
...
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: "smtp.zoho.com",
+    port: 465,
+    domain: "memrybook.com",
+    authentication: :login,
+    tls: true,
+    enable_starttls_auto: true,
+    user_name: "info@memrybook.com",
```
Comment (allanxavierm) [inline config/environments/development.rb]: Move all these credentials into environment variables, make use of https://github.com/bkeepers/dotenv for it. What about in the production environment ?

File: app/mailers/user_mailer.rb (hunk 1 above)
Comment (basantrijal) [inline app/mailers/user_mailer.rb:5]: registration success template is used for mail templates and i need to pass email there.

**Pull request merged**